data/sql/updates/pending_db_world/rev_1550009247084015800.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1550009247084015800');
+
+-- Originally from Shin
+-- creature_template https://github.com/azerothcore/azerothcore-wotlk/pull/1322/files
+ALTER TABLE `creature_template`
+  CHANGE `baseattacktime` `BaseAttackTime` INT(10) UNSIGNED DEFAULT 0  NOT NULL,
+  CHANGE `rangeattacktime` `RangeAttackTime` INT(10) UNSIGNED DEFAULT 0  NOT NULL,
+  CHANGE `Health_mod` `HealthModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `Mana_mod` `ManaModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `Armor_mod` `ArmorModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `dmg_multiplier` `DamageModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `VerifiedBuild` `VerifiedBuild` SMALLINT(5) DEFAULT '0';
```

Comment:
this change is already included here: https://github.com/azerothcore/azerothcore-wotlk/blob/master/data/sql/updates/db_world/2019_01_20_02.sql

Reply:
Mhh it was still giving me an error about it. After a clean dB install and clean pull.